### PR TITLE
chore: make `help` tab into module

### PR DESCRIPTION
**Pull request by chmelevskij**
`help` tab is a module now.

### CODE REVIEW

File: src/js/utils/common.js [hunk 1]
```diff
@@ -76,7 +76,6 @@ export function generateVirtualApiVersions() {
 }
 
 // TODO: these are temp binding while transition to module happens
-window.microtime = microtime;
```
Comment (McGiverGim) [inline src/js/utils/common.js:79]: I suppose this is not related to this PR and this is being fixed here: https://github.com/betaflight/betaflight-configurator/pull/2414

**Review by McGiverGim — CHANGES_REQUESTED** [review]

File: src/js/utils/common.js (hunk 1 above)
Comment (chmelevskij) [reply src/js/utils/common.js:79]: true, but I can't just push broken code (as I did). So will rebase when that is done. But till then I'll keep this here.

**Review by mikeller — COMMENTED** [review]
I think this can be rebased now.

**Review by mikeller — APPROVED** [review]

**Comment by chmelevskij** [comment]
Closing in favour of #2430
